core/src/test/scala/pureconfig/ApiSuite.scala

```diff
@@ -128,9 +121,16 @@ class ApiSuite extends BaseSuite {
     loadConfigFromFiles[Conf](files) should failWithType[ThrowableFailure]
   }
 
-  it should "fail if any of the files does not exist" in {
-    case class Conf(f: Float)
-    val files = listResourcesFromNames("/conf/loadConfigFromFiles/priority1.conf") :+ nonExistingPath
-    loadConfigFromFiles[Conf](files) should failWithType[CannotReadFile]
+  it should "ignore files that doesn't exist" in {
+    case class Conf(b: Boolean, d: Double)
+    val files = listResourcesFromNames("/conf/loadConfigFromFiles/priority2.conf") :+ nonExistingPath
+    loadConfigFromFiles[Conf](files) shouldBe Right(Conf(false, 0.001D))
+  }
+
+  "loadConfigWithFallback" should "fallback if no config keys are found" in {
+    case class Conf(f: Float, o: Option[Int], d: Double)
+    val priority1Conf = ConfigFactory.load("conf/loadConfigFromFiles/priority1.conf")
+    // first wo are in priority1.conf, the d is in reference.conf
```

Comment:
The amusing bit that the original typo is almost certainly mine. :-)
Thanks for fixing that.